app/[locale]/layout.tsx

```diff
@@ -24,7 +24,7 @@ const RootLayout: FC<PropsWithChildren> = ({ children }) => {
 
   return (
     <html className={sourceSans.className} dir={langDir} lang={hrefLang}>
-      <body>
+      <body suppressHydrationWarning>
```

Comment:
TIL about this https://legacy.reactjs.org/docs/dom-elements.html#suppresshydrationwarning